packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/getuserprofile.md

```diff
@@ -1,27 +0,0 @@
-## Get the user profile information
```

Comment:
Removed as it is an optional step in the guide. I checked for links to the section in other pages and didn't find any, should be ok to remove.